src/utils.ml

```diff
@@ -201,6 +201,7 @@ module Cached_digest = struct
   type file =
     { mutable digest            : Digest.t
     ; mutable timestamp         : float
+    ; mutable permissions       : Unix.file_perm
     ; mutable timestamp_checked : int
```

Comment:
This should no longer be called `timestamp_checked` given that we also check the permissions. What about `stats_checked`?

Reply:
done